app/src/main/res/values/dimens.xml

```diff
@@ -211,4 +211,6 @@
     <dimen name="accessibility_min_height">48dp</dimen>
 
     <dimen name="tap_increase_16">16dp</dimen>
+
+    <dimen name="home_collection_elevation">5dp</dimen>
```

Comment:
There's a `<!-- Home Fragment -->` on line 93. Should we just throw this up there?

Reply:
moved!